portfolio/src/main/webapp/index.html

```diff
@@ -9,7 +9,7 @@
 
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments()">
```

Comment:
Can `onload="getComments()"` be moved below to `<ul id="comments-container">`?

Reply:
I tried adding the onload to the comments container, but it didn't seem to work. I tried it when the container was a div, but I'll give it another shot now that it's a ul :)